Assets/MRTK/Examples/Experimental/StepSlider/StepSliderExample.unity

```diff
@@ -0,0 +1,8073 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!29 &1
+OcclusionCullingSettings:
+  m_ObjectHideFlags: 0
+  serializedVersion: 2
+  m_OcclusionBakeSettings:
+    smallestOccluder: 5
```

Comment:
It looks like the main camera in this scene is missing a script, when I test this branch:

![image](https://user-images.githubusercontent.com/168492/92665133-a12ad880-f2ba-11ea-9857-b4a2b52ce5ce.png)

Reply:
I'm not sure why this is the case, I can see the script in my scene. I'll make sure to do some branch testing on different machines.
![image](https://user-images.githubusercontent.com/15256397/92668017-75135580-f2c2-11ea-97f5-dcc3b438717a.png)

Reply:
It looks like this might have been an artifact of me using Unity 2019.4, where many other things were broken. I'll resolve this for now.

Reply:
> this might have been an artifact of me using Unity 2019.4

@gauravb4 which version of Unity did you use to build this?

Reply:
Built on 2019.4.7f1, tested on 2019.3.9f1, both had the script. Going to be testing on Mac today.